filters/include/pcl/filters/impl/crop_hull.hpp

```diff
@@ -113,16 +102,44 @@ pcl::CropHull<PointT>::getHullCloudRange ()
       if (pt[i] > cloud_max[i]) cloud_max[i] = pt[i];
     }
   }
-  
+
   return (cloud_max - cloud_min);
 }
 
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 template<typename PointT> template<unsigned PlaneDim1, unsigned PlaneDim2> void 
 pcl::CropHull<PointT>::applyFilter2D (PointCloud &output)
 {
+  std::vector<int> indices;
```

Comment:
No need to clear the output like in L47?

Reply:
`applyFIlter2D` and `applyFilter3D` are private functions, so only `applyFilter` can call `applyFilter[23]D`.